frame/collective/src/lib.rs

```diff
@@ -669,11 +693,30 @@ pub mod pallet {
 	#[pallet::genesis_build]
 	impl<T: Config<I>, I: 'static> GenesisBuild<T, I> for GenesisConfig<T, I> {
 		fn build(&self) {
+			<UpgradedToTripleRefCount<T, I>>::put(false);
 			Pallet::<T, I>::initialize_members(&self.members);
 		}
 	}
 }
 
+mod migrations {
+	use super::*;
+
+	pub fn migrate_to_triple_ref_count<T: Config<I>, I: 'static>() -> frame_support::weights::Weight {
```

Comment:
name should be more meaningful, maybe `migrate_to_new_pallet_prefix`

Reply:
Make sense, Done :+1: